uninstaller/uninstall-helper.ps1

```diff
@@ -72,7 +72,12 @@ $pathFolders | foreach {
 [string] $delimitedFolders = $folderList -Join ';'
 [Environment]::SetEnvironmentVariable("Path", $delimitedFolders, "User")
 
-Remove-Item -Path "HKLM:\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\DevelopmentSuite$timeStamp"
+for ($i=0; $i -le $key.Count; $i++) {
```

Comment:
This would be definitely data loss in case there are several devsuite installs on the same machine.

Reply:
This would not work for scenario:
1. Install DevSuite into `c:\Program Files\DevelopmentSuite';
2. Install DevSuite into `c:\Program Files\DevelopmentSuiteAgain';
3. You have two entries in control panel;
4. Uninstall one install;
5. Check if another one is still there. If both are gone you have a problem.